app/components/project-post-list.js

```diff
@@ -28,10 +28,10 @@ export default Ember.Component.extend({
   _normalizeMeta(meta) {
     if (Ember.isPresent(meta)) {
       return {
-        currentPage: parseInt(meta.current_page, 10),
-        pageSize: parseInt(meta.page_size, 10),
-        totalPages: parseInt(meta.total_pages, 10),
-        totalRecords: parseInt(meta.total_records)
+        currentPage: parseInt(meta["current-page"], 10),
+        pageSize: parseInt(meta["page-size"], 10),
+        totalPages: parseInt(meta["total-pages"], 10),
+        totalRecords: parseInt(meta["total-records"])
```

Comment:
Okay, are you able to pull down this branch at all and see what else I'm missing, there are still some failures that I'm stepping through now.